src/dev/lupio/LupioRTC.py

```diff
@@ -0,0 +1,19 @@
+from m5.params import *
+from m5.proxy import *
+from m5.objects.Device import BasicPioDevice
+from m5.util.fdthelper import *
+
+class LupioRTC(BasicPioDevice):
+    type = 'LupioRTC'
+    cxx_class='gem5::LupioRTC'
+    cxx_header = 'dev/lupio/lupio_rtc.hh'
+    time = Param.Time('01/01/2012',
+        "System time to use")
```

Comment:
Does this need to be on two lines? It doesn't seem like the line would be over 80 characters.